src/lib.rs

```diff
@@ -168,7 +202,41 @@ pub async fn run(config: &Config) -> Result<(), ()> {
         Some(s) => s.iter().map(|m| WildMatch::new(m)).collect(),
         None => vec![],
     };
+
+    let gitignored_files: Option<Vec<PathBuf>> = if config.optional.gitignore.is_some() {
+        let files = find_git_ignored_files();
+        println!("Found gitignored files: {:?}", files);
```

Comment:
either use ìnfo!` or `debug!` for this log line I guess

Reply:
Yeah this was for my own debugging actually. However I left it in with a `debug!` in case it's useful for anyone else in the future.